cli/commands.js

```diff
@@ -49,6 +49,9 @@ module.exports = {
         argument: ['languages...'],
         desc: 'Select languages from a list of available languages. The i18n files will be copied to the /webapp/i18n folder'
     },
+    linker: {
```

Comment:
i'm not sure if its good name for this, `link` or `link-account` might be better

Reply:
But in general I dont think we need a sub-gen for this

Reply:
What should I do ? Put it in the `app` subgen ?

Reply:
Ok lets keep the sub-gen but rename it to `link-account` and we can have one more question after opt-in asking `Would you like to link your JHipster Online account, to get access to and manage your own stats` if the user says yes compose this sub-gen there and get credentials and do the link